packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -218,11 +345,14 @@ export const makeVaultManager = (
 
   observeNotifier(periodNotifier, timeObserver);
 
-  /** @type {InnerVaultManager} */
-  const innerFacet = harden({
+  /** @type {Parameters<typeof makeVaultKit>[1]} */
+  const managerFacade = harden({
```

Comment:
Is this something other than what we call a facet?

```suggestion
  const managerFacet = harden({
```